### ISSUE TITLE
Starfield gpu usage not reseting after changing graphics with staggered submit on.

### ISSUE THREAD

**Issue by Hadrianneue** [HansKristian-Work/vkd3d-proton#2193]
Game greatly benefits from staggered submit, while it doesn't improve FSR3 frame generation, it smooths out greatly the frametime graph with or without frame generation on.

However, with staggered submit on, changing settings, say graphics preset from low to high and vice-versa, we get a permanent GPU performance/usage drop, it doesn't reset usage/performance, requiring closing and launching the game again, that doesn't happen with the  no_staggered_submit flag.

### Software information
KDE x11/wayland - Linux 6.11.5

### System information
- GPU: RX 6700 XT
- Driver: Mesa-git
- Wine version: Proton Experimental bleeding-edge
- VKD3D-Proton version: 2.13-bleeding-edge

### Log files

[steam-1716740.log](https://github.com/user-attachments/files/17651177/steam-1716740.log)

**Comment by oabdellatif** [comment]
I’ve also observed this on Proton Experimental/GE-Proton9-20. With the framerate uncapped the GPU usage is close to 99% on a fresh launch, but drops to 80-90% after changing any graphics settings (RX 7800 XT).

Importantly, I also notice the same drop with FSR 3 FG enabled regardless of changing any settings. FG performance is rather poor as well (only a 50% improvement roughly). @Hadrianneue Have you observed similar results?

**Comment by Hadrianneue** [comment]
> I’ve also observed this on Proton Experimental/GE-Proton9-20. With the framerate uncapped the GPU usage is close to 99% on a fresh launch, but drops to 80-90% after changing any graphics settings (RX 7800 XT).
> 
> Importantly, I also notice the same drop with FSR 3 FG enabled regardless of changing any settings. FG performance is rather poor as well (only a 50% improvement roughly). @Hadrianneue Have you observed similar results?

yes, that's because of a limitation in RADV currently, its not exclusive to starfield, some work was done in VKD3D to mitigate that, and it does help but does not fix it, you can disable it with VKD3D_CONFIG=no_staggered_submit, but your frametimes will surfer greatly.
